core/src/main/scala/org/http4s/ServerSentEvent.scala

```diff
@@ -30,7 +30,9 @@ final case class ServerSentEvent(
     retry: Option[Long] = None
 ) extends Renderable {
   def render(writer: Writer): writer.type = {
-    writer << "data: " << data << "\n"
+    data.linesIterator.foreach { dataLine =>
```

Comment:
I brought this up on the other ticket.  `linesIterator` will split on `\r`, `\r\n`, and `\n`.  This renderer will reassemble it with just `\n`, so it's a lossy conversion.  It's legal to separate the lines with `\r` or `\r\n`, so it doesn't _have_ to be lossy.

This is probably an exotic case, and what you've done is inarguably an improvement for real-world use.